src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -24,25 +24,43 @@ object Run  {
   val PollingBackoffFactor = 1.1
 
   def apply(pipeline: Pipeline): Run = {
-    val rpr = new RunPipelineRequest().setPipelineId(pipeline.id).setProjectId(pipeline.projectId).setServiceAccount(JesServiceAccount)
     val tag = s"JES Run [UUID(${pipeline.workflow.shortId}):${pipeline.key.tag}]"
+    if (!(pipeline.pipelineId.isDefined ^ pipeline.runIdForResumption.isDefined)) {
```

Comment:
Isn't this the same as `if (pipeline.pipelineId.isDefined == pipeline.runIdForResumption.isDefined)`
Btw I didn't know that there is a `XOR` operator in scala !